bitnami/nginx/templates/deployment.yaml

```
@@ -90,7 +90,8 @@ spec:
             - -ec
             - |
               [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
-              git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /app
+              git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /tmp/app
+              [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/repo/* /app/
```

It is not exactly a copy, please check the diff here:

```suggestion
              git clone {{ .Values.cloneHtdocsFromGit.repository }} --branch {{ .Values.cloneHtdocsFromGit.branch }} /tmp/repo
              [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/repo/* /app/
```

You replaced `/tmp/repo` with `/tmp/app` in the first line, but not in the second.
That way, you are cloning the repository into `/tmp/app` and never using it, and moving a the folder `/tmp/repo` which was never created.

---

Looks like the `/tmp/repo` vs `/tmp/app` conflict appeared again, could you please fix this again?